### DISCUSSION TITLE
Possible bug? ClickOptions typescript

### DISCUSSION THREAD

**Discussion by tomtomdurrant** [cypress-io/cypress#9167] [Questions and Help] [answered]
Hi, I'm trying to control click multiple elements using the `.click({multiple: true, ctrlKey: true})` but the type definition does not contain the any properties for additional keys such as control or shift etc. When I add that property to my own custom interface it seems to not work in the test runner. 

Is there something I'm missing?

**Comment by sainthkh** [comment, marked as answer]
They do exist. Check the code below:

https://github.com/cypress-io/cypress/blob/b08817bd868baaaf9d3a6e825af229a029fce475/cli/types/cypress.d.ts#L2369-L2374

**Reply by tomtomdurrant** [reply]
Just realised I'm on an old version. Thanks 🤦‍♂️

**Comment by jennifer-shehane** [comment]
@tomtomdurrant Make sure you're also using the most current version of Cypress. These new options were introduced in 5.0.0

**Reply by tomtomdurrant** [reply]
Currently on an old version. That will be the problem. Thanks 😂
